nixos/maintainers/scripts/ec2/amazon-base-config.nix

```diff
@@ -1,5 +1,5 @@
-{ modulesPath, ...}:
+{ ... }:
 {
-  imports = [ "${modulesPath}/virtualisation/amazon-config.nix" ];
```

Comment:
This doesn't seem like the right change. I thought you were adding `modulesPath` back in, so why is this needed? Same with azure. Using `<nixpkgs>` there assumes that this file only gets copied into the target filesystem, whereas some image building code also uses it.